x/evm/keeper/grpc_query.go

```diff
@@ -0,0 +1,29 @@
+package keeper
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/axelarnetwork/axelar-core/x/evm/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+var _ types.QueryServiceServer = baseKeeper{}
+
+func (k baseKeeper) BurnerInfo(c context.Context, req *types.BurnerInfoRequest) (*types.BurnerInfoResponse, error) {
+	ctx := sdk.UnwrapSDKContext(c)
+
+	if !k.getBaseStore(ctx).Has(subspacePrefix.AppendStr(strings.ToLower(req.Chain))) {
```

Comment:
Can you define a method in the `baseKeeper` for checking if a chain exists in the kvstore?

Reply:
i guess

Reply:
done